pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -721,13 +723,6 @@ func waitUntilDNSReady(zedagentCtx *zedagentContext, stillRunning *time.Ticker)
 		case change := <-zedagentCtx.subLocationInfo.MsgChan():
 			zedagentCtx.subLocationInfo.ProcessChange(change)
 
-		case change := <-zedcloudCtx.DeferredEventCtx.Ticker.C:
-			start := time.Now()
-			zedcloudCtx.DeferredEventCtx.HandleDeferred(
-				change, 100*time.Millisecond, false)
-			zedagentCtx.ps.CheckMaxTimeTopic(agentName, "deferredEventCtx", start,
-				warningTime, errorTime)
-
```

Comment:
At a minumum this needs to be added to handleDeferredTask() and then retested, but I'd feel a lot more confortable with the set of changes in the path I emailed (apart from the "if true else XXX" which should be removed. I can push that to the PR branch if it makes sense.